### PR TITLE
ARUHA-1560 Update documentation

### PR DESCRIPTION
**Pull request by lmontrieux**
# Make architecture document public

> Zalando ticket : ARUHA-1560

## Description
There was a link to an internal document, and we decided to make the document public

### CODE REVIEW

**Comment by lmontrieux** [comment]
👍

File: docs/_documentation/intro.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ The goal of Nakadi (**ნაკადი** means "stream" in Georgian) is to pro
 ### Links
 
 Read more to understand *The big picture* 
-[Architecture for data integration](https://pages.github.bus.zalan.do/core-platform/docs/architecture/data_integration.html) 
+[Architecture for data integration](https://zalando.github.io/nakadi/manual.html#data_integration_at_zalando) 
 
 Watch the talk [Data Integration in the World of Microservices](https://clusterhq.com/2016/05/20/microservices-zalando/)
```
Comment (SergKam) [inline docs/_documentation/intro.md]: Unfortunately this link also gone :( 
Please change it to https://www.youtube.com/watch?v=SbVQBHRAFXA

Comment (lmontrieux) [reply docs/_documentation/intro.md]: It's not gone, it's where the new page should be :)

Comment (SergKam) [reply docs/_documentation/intro.md]: Didn't get the joke :confused: Please just change the link to the video :pray: .

Comment (lmontrieux) [reply docs/_documentation/intro.md]: It's not a joke, it is the location of the new page generated by the previous file. But I can add the video as well.

Comment (lmontrieux) [reply docs/_documentation/intro.md]: dfc0836

**Comment by lmontrieux** [comment]
👍

**Comment by SergKam** [comment]
:+1:

**Comment by antban** [comment]
:+1:

**Pull request merged**